compiler/src/dotty/tools/dotc/transform/PickleQuotes.scala

```diff
@@ -103,9 +103,7 @@ class PickleQuotes extends MacroTransform {
           val pickled = PickleQuotes.pickle(quote1, quotes, contents)
           transform(pickled) // pickle quotes that are in the contents
         case tree: DefDef if !tree.rhs.isEmpty && tree.symbol.isInlineMethod =>
-          // Shrink size of the tree. The methods have already been inlined.
-          // TODO move to FirstTransform to trigger even without quotes
-          cpy.DefDef(tree)(rhs = defaultValue(tree.rhs.tpe))
+          tree
```

Comment:
We only removed the bodies of inline methods in a file containing a quote. All other inline methods kept the contents of inlined definitions until `PruneErasedDefs`.